AWSiOSSDKv2.xcodeproj/xcshareddata/xcschemes/AWSAllTests.xcscheme

```diff
@@ -1866,6 +1760,15 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES">
+      <MacroExpansion>
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "CE9DEB1D1C6A81160060793F"
+            BuildableName = "AWSAPIGateway.framework"
+            BlueprintName = "AWSAPIGateway"
+            ReferencedContainer = "container:AWSiOSSDKv2.xcodeproj">
+         </BuildableReference>
+      </MacroExpansion>
```

Comment:
Not sure why this was added, but assuming it's okay.

Reply:
I wish I understood why this happens: I have removed it in the past and it keeps showing back up. I'll remove it in a separate PR, since this one is so...weighty. :)

Reply:
Re-reviewing this, it seems innocuous, so I'm going to leave it in place. I'm not sure why it wasn't present before but the AWSAllTests scheme enables the AWSAPIGatewayUnit tests, which would require the SDK itself to be built. It built in the pipeline to test this PR, but if I see failures in the future I'll re-visit this.